### PR TITLE
Migrated notebook to Vertex AI Workbench instance in data-playground blueprints

### PR DESCRIPTION
**Pull request by gioconte**
Updating this blueprinting by migrating the notebook to [Vertex AI workbench instance](https://cloud.google.com/vertex-ai/docs/workbench/instances/introduction), since vertex ai workbench user-managed/managed notebook are [deprecated](https://cloud.google.com/vertex-ai/docs/deprecations)

---
**Checklist**

I applicable, I acknowledge that I have:
- [X] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [X] Ran `terraform fmt` on all modified files
- [X] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [X] Made sure all relevant tests pass

### CODE REVIEW

File: blueprints/data-solutions/data-playground/main.tf [hunk 1]
```diff
@@ -189,43 +190,53 @@ module "service-account-notebook" {
       "roles/bigquery.user",
       "roles/dialogflow.client",
       "roles/storage.admin",
+      "roles/serviceusage.serviceUsageConsumer",
+      "roles/aiplatform.user",
+      "roles/viewer"
```
Comment (wiktorn) [inline blueprints/data-solutions/data-playground/main.tf:195]: Can you replace this basic role with more specific role that is required?

File: blueprints/data-solutions/data-playground/main.tf [hunk 2]
```diff
@@ -74,7 +74,8 @@ module "project" {
     "servicenetworking.googleapis.com",
     "stackdriver.googleapis.com",
     "storage.googleapis.com",
-    "storage-component.googleapis.com"
+    "storage-component.googleapis.com",
+    "dataproc.googleapis.com"
```
Comment (wiktorn) [inline blueprints/data-solutions/data-playground/main.tf:78]: Any specific reason to activate `dataproc` service? Is it mandatory to run this blueprint?

**Review by wiktorn — COMMENTED** [review]
Thank you very much for this.

I guess we need to do the same for [Vertex MLOps blueprint](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/blueprints/data-solutions/vertex-mlops/vertex.tf#L82)

**Comment by lcaggio** [comment]
+1 on doing the same on the [Vertex MLOps blueprint](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/blueprints/data-solutions/vertex-mlops/vertex.tf#L82) blueprint. 

What about deprecating this in favor of the other more generic one? which use case this blueprint is addressing in addition to the MLOps one?

**Comment by ludoo** [comment]
@gioconte this has been idle since it was opened. Do you still plan on merging this, or shall we close it?

**Comment by ludoo** [comment]
Closing due to lack of traction on review comments.

**Pull request closed without merging**